tests/policyTest/a/repositoryCfg.yaml

```diff
@@ -0,0 +1,9 @@
+!RepositoryCfg_v1
+_mapper: !!python/name:lsst.obs.test.testMapper.TestMapper ''
+_mapperArgs: null
+_parents: []
+_policy: !!python/object:lsst.daf.persistence.policy.Policy
+  data:
+    exposures:
+      raw: {template: raw/v%(visit)d_f%(filter)s.fits.gz}
```

Comment:
I didn't even think about it. 2 spaces seems to be the default for `yaml.dump`. I *can* specify the indentation level when I call `dump` though.
@timj is 2 spaces ok or should I specify 4?